engine.py

```diff
@@ -249,7 +274,6 @@ def __getattr__(self, name):
                                      "run UI applications from Houdini.")
 
         base = {"qt_core": QTProxy(), "qt_gui": QTProxy(), "dialog_base": None}
-        self._ui_type = None
```

Comment:
Without this line wouldn't there be a member variable accessed before it is defined?